src/arch/riscv/kernel/vspace.c

```diff
@@ -136,7 +135,7 @@ BOOT_CODE VISIBLE void map_kernel_window(void)
     kernel_root_pageTable[RISCV_GET_PT_INDEX(KS_LOG_PPTR, 0)] =
         pte_next(kpptr_to_paddr(kernel_image_level2_log_buffer_pt), false);
 #endif
-#else
+#else /* __riscv_xlen != 32 */
```

Comment:
This comment is ambiguous, it could mean that the following block is for the `!=` case, or it could mean that the `!=` is completed, and the `else` case (the `==` case) follows. Having no comment here was better. I'd be fine with just `/* __riscv_xlen */` as comment, then at least there is no ambiguity.

Reply:
(of course it's clear which case it is when you scroll up to look at the code, but then you don't need a comment)